lib/msf/core/modules/metadata/maps.rb

```diff
@@ -0,0 +1,69 @@
+#
+# Core service class that provides storage of module metadata as well as operations on the metadata.
+# Note that operations on this metadata are included as separate modules.
+#
+module Msf::Modules::Metadata::Maps
+
+  @mrefs  = {}
+  @mports = {}
+  @mservs = {}
+
+
+  def all_remote_exploit_maps
+
+    unless @mrefs
```

Comment:
The entire method, except the `return`, is wrapped in this `unless` conditional. Consider a guard clause.

```ruby
  return @mrefs, @mports, @mservs unless @mrefs
```